db/re/skill_db.yml

```diff
@@ -43509,25 +43507,25 @@ Body:
     AfterCastActDelay: 3000
     Duration2:
       - Level: 1
-        Time: 30000
+        Time: 28000
       - Level: 2
-        Time: 35000
+        Time: 33000
       - Level: 3
-        Time: 40000
+        Time: 38000
       - Level: 4
-        Time: 45000
+        Time: 43000
       - Level: 5
-        Time: 50000
+        Time: 48000
       - Level: 6
-        Time: 60000
+        Time: 58000
       - Level: 7
-        Time: 60000
+        Time: 58000
       - Level: 8
-        Time: 60000
+        Time: 58000
       - Level: 9
-        Time: 60000
+        Time: 58000
       - Level: 10
-        Time: 60000
+        Time: 58000
```

Comment:
Probably should remain unchanged (it's not using the default duration), same as Lex Divina.